example.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.14"
+}
+
+module "my_vpc" {
+  source = "./modules/vpc"
```

Comment:
Input values should be explicitly specified.
If you rely on defaults, there would be a high chance of human error when deploying complex infrastructures.